internal/types/plugin.go

```diff
@@ -0,0 +1,22 @@
+package types
+
+import "encoding/json"
+
+type Plugin struct {
+	ID             string          `json:"id" validate:"required"`
+	Title          string          `json:"title" validate:"required"`
+	Description    string          `json:"description"`
+	Metadata       json.RawMessage `json:"metadata"`
+	ServerEndpoint string          `json:"server_endpoint"`
+	Vaults         string          `json:"vaults"`
```

Comment:
this corresponds to a single vault(pubkey), so lets just rename it to `PublicKey`

Reply:
:+1: Changed [here](https://github.com/LimeChain/vultiserver-plugin/pull/40/commits/dfb9c1ee6e0cb3ecf077f736e5f8aa57b1baed1c)

Reply:
Removed as it would be a relation with all users that installed the plugin
https://github.com/LimeChain/vultiserver-plugin/pull/40/commits/b6c84784e094cef84072b8695b67143384f58c93